server/controllers/post-form-controller.js

```diff
@@ -0,0 +1,17 @@
+const main = (req, res) => {
+	console.log('req.body.name:', req.body.name);
+	console.log('req.body.email:', req.body.email);
+	console.log('req.body.country:', req.body.country);
+	console.log('req.body.gdpr:', req.body.gdpr);
```

Comment:
A nice feature of ES2015+ is destructuring. You can read about it from Wes here https://wesbos.com/destructuring-objects/

Destructuring your `res` object like:

```js
const main = ({ body: { name, email, country, gdpr }}, res) => {
```
will allow you to reference those attributes directly in the code as:
```
console.log('name:', name);
```

Reply:
Ooh. I'm going to try that out now. I like the idea of writing less code but getting it to do more!